### PR TITLE
Create _vercel.alora.json

### PR DESCRIPTION
**Pull request by aloramiaa**
Just Vercel Txt Verfiication. Cname has been set Already in profile.alora.json and drive.alora.json

### CODE REVIEW

File: domains/_vercel.alora.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "aloramiaa",
+    "email": "xaloramia@gmail.com",
+    "discord": "1236292707371057216"
+  },
+  "record": {
+    "TXT": "vc-domain-verify=drive.alora.is-a.dev,395400702612ae6db6a8"
```
Comment (DEV-DIBSTER) [inline domains/_vercel.alora.json:8]: TXT records should be in an array as opposed to two objects which is invalid JSON.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by aloramiaa** [comment]
Updated

File: domains/_vercel.alora.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/_vercel.alora.json]: It works even in a string though?

Comment (CuteDog5695) [reply domains/_vercel.alora.json]: Nvm just realised its 2 strings

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**